### ISSUE TITLE
Block explorer

### ISSUE THREAD

**Issue by i-norden** [cerc-io/laconicd#29]
From Sjors: Would recommend using https://github.com/forbole/big-dipper-2.0-cosmos or https://github.com/ping-pub/explorer. Would be good to have this already in the testnet phase.

**Comment by AFDudley** [comment]
Should this issue be moved to stack orchestrator? It was suggested that we use ping-pub's until work resumes on ours.

**Comment by i-norden** [comment]
@AFDudley yeah I think there or ops or issue_tracking would be more appropriate since it does not entitle implementation work in this repository
